first-party-dependencies/pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
License & Copyright?

Reply:
I don't find Yoshi Java repositories declare license and copyright in pom.xml files. I see `<organization>` and `<license>` elements, which I think should deliver License & Copyright.